tests/acceptance/features-test.js

```diff
@@ -13,6 +15,16 @@ module('Acceptance: Features', {
   }
 });
 
+test('features are defined in app config', function() {
+  ok(features.enabled('feature-from-config'), 'feature-from-config is enabled automatically');
```

Comment:
That assertion tests that the features module is correctly configured when the app starts up, so I think it does. Have removed it anyway.